drakrun/drakrun/draksetup.py

```diff
@@ -186,6 +186,10 @@ def sanity_check():
         logging.error("Your Xen installation doesn\'t pass the necessary tests.")
         return False
 
+    # redis sanity check
+    logging.info("Checking redis")
+    redis_working(conf)
+
```

Comment:
The `draksetup` is completely unrelated to Redis itself, so I think it shouldn't be there.

Reply:
Umm, then what am I supposed to do in the second point mentioned in the issue?